install_bootstrap

```diff
@@ -189,18 +189,30 @@ while [[ "${install_dockstore_launcher^^}" != 'Y' &&  "${install_dockstore_launc
 				read user_install_docker
 			done
 		fi
-                echo "Installing various apt packages"
-                # need to install regardless
-		set +e
-		sudo apt-get install wget curl ruby-mustache=1.0.2-1 jq git --yes &> install_wget_curl.log
-		install_wget_result=$?
-		set -e
+    echo "Installing various apt packages"
+    # need to install regardless
+    if ! command -v jq &> /dev/null
+    then
+        sudo snap install jq
+    fi
+		if ! command -v wget &> /dev/null
```

Comment:
Formatting is off, but if you just remove this (Gary's comment about installing wget twice)